R/utils_accessibility.R

```diff
@@ -126,53 +126,65 @@ nhs_card_tabstop <- function(header, tabindex = 0, ...) {
 }
 
 
-
-
-
-
-
 #' accessible_radio_button
 #'
-#' @param ... radio content
+#' @inheritParams shiny::radioButtons
```

Comment:
Easy way to get the parameter documentation from the `shiny` function it is based off.